doc/flame_audio/audio.md

```diff
@@ -7,7 +7,7 @@ list in your `pubspec.yaml` file:
 
 ```yaml
 dependencies:
-  flame_audio: <VERSION>
+  flame_audio: --VERSION--
```

Comment:
This one is actually wrong, it was wrong before you changed it though, because this is not tied to the Flame version.
Maybe just write VERSION here?

Reply:
Ok

Reply:
No, because other file is talking about Flame, and `--VERSION--` corresponds to the Flame version, so that will show the correct version. :)